### ISSUE TITLE
switchpath return type mismatch

### ISSUE THREAD

**Issue by kylecordes** [cyclejs-community/cyclic-router#192]
switchpath returns this:

```
export interface SwitchPathReturn {
    path: string | null;
    value: any | null;
}
```

while cyclic expects this:

```
export interface RouteMatcherReturn {
    path: string;
    value: any;
}
```

I'm not deep enough in to know whether this is just a matter of adjusting the type in cycle to keep up with switchpath, or if the possible null return requires more changes.

**Comment by SteveALee** [comment]
+1 as it would be good to [lose this cast](https://github.com/cyclejs-community/one-fits-all/blob/6afb2c729974fbb2c16eca630cd6226f2a4b801f/template/src/drivers.ts#L47) - given switch-path is default

**Comment by jvanbruegge** [comment]
Will fix that later, easy solution

**Comment by jvanbruegge** [comment]
Published in 5.1.1
